### PR TITLE
feat(appBridge): Add proxyNetworkCall method (#821)

### PR DESCRIPTION
**Pull request by imoutaharik**
* feat(PlatformApp): add new API method

* Update .changeset/clever-gifts-join.md

---------

### CODE REVIEW

**Review by marguslant0 — APPROVED** [review]
LGTM! 🎉

**Review by anxobotana — APPROVED** [review]
Owned files look good

**Review by ragi96 — APPROVED** [review]
Owned files are looking good

File: .changeset/four-cherries-beg.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (ragi96) [inline .changeset/four-cherries-beg.md:1]: Remove that changeset, probably better to use a dedicated branch then `main` -> `dev` for the merge, as here could happen some issues

**Review by ragi96 — CHANGES_REQUESTED** [review]

**Comment by imoutaharik** [comment]
Closing this one and will do differently.

**Pull request closed without merging**